api/tests/prescription/learner-management/integration/domain/usecases/has-been-learner_test.js

```diff
@@ -0,0 +1,43 @@
+import { usecases } from '../../../../../../src/prescription/learner-management/domain/usecases/index.js';
+import { databaseBuilder, expect } from '../../../../../test-helper.js';
+
+describe('Integration | Prescription | Learner Management | UseCase | has-been-learner', function () {
+  let userId;
+
+  beforeEach(async function () {
+    const organizationId = databaseBuilder.factory.buildOrganization().id;
+    const otherOrganizationId = databaseBuilder.factory.buildOrganization().id;
+
+    userId = databaseBuilder.factory.buildUser().id;
+    const otherUserId = databaseBuilder.factory.buildUser().id;
+
+    databaseBuilder.factory.buildOrganizationLearner({ organizationId, userId });
+    databaseBuilder.factory.buildOrganizationLearner({ organizationId, userId: otherUserId });
+    databaseBuilder.factory.buildOrganizationLearner({ organizationId: otherOrganizationId, userId });
+
+    await databaseBuilder.commit();
+  });
+
+  context('when organization learner exist for given userId', function () {
```

Comment:
**suggestion** j'aurai fait un seul it d'integration. avec une orga pour avoir un compte . 

là j'ai l'impression qu'on fait la même chose que pour le repo le repository au final 🤔 .

Et si on cherche à savoir que le résultat retourner et bien celui du repo dans ce cas simple. faire un TU qui valide que le retour c'est bien ce que le repo retourne ?

Reply:
Bonne remarque ! 

C'est vrai que les données de test sont les même que ceux du repository. Vais remplacer ça par un TU 😄

Reply:
on peut laisser un test d'integ même si il retourne 0 . pour vérifier que le branchement est correct ( injection de dépendance etc...) . mais effectivement un TU qui valide que le usecase retourne simplement le contenu du repo pourrait suffire.